examples/circle_packing.ml

```diff
@@ -2,6 +2,7 @@ open Joy
 
 (* global constants // RNG initialization *)
 let resolution = (1200., 900.)
+let tmap f (x, y) = (f x, f y)
```

Comment:
If it were ints I would need to add 5-6 more calls to `float_of_int`, as opposed to the 1 call to `tmap int_of_float resolution` that's present.

Reply:
Where would that be required?

Reply:
Line 40 in `rand_circle`. I think this example needs to be rewritten from scratch though, it doesn't properly pack the circles since migrating to the Cairo backend and I think it could be made a bit more concise.